app/views/posts/index.html.erb

```diff
@@ -1,2 +1,14 @@
-<h1>All Posts</h1>
-<p>This is a placeholder for posts index content.</p>
+<ul class="space-y-4">
```

Comment:
- [ ] @aradradev Great work  on your posts page, however the page is supposed to have details about the author and the number of posts at the top of the page, check the given design below to understand it better, also the pagination button is missing
![image](https://github.com/aradradev/Blog-app/assets/61408860/e66667f9-5211-4fe6-9310-8f93197f5325)